easybuild/easyconfigs/h/HBase/HBase-1.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'HBase'
+version = '1.0.2'
+
+homepage = 'http://hbase.apache.org/'
+description = """Apache HBase. is the Hadoop database, a distributed, scalable, big data store. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin.tar.gz']
+source_urls = ['http://www.us.apache.org/dist/hbase/hbase-1.0.2/']
```

Comment:
please use `%(version)s` here too, to avoid having the version hard-coded in multiple places:

```
source_urls = ['http://www.us.apache.org/dist/hbase/hbase-1.0.2/']
```